application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadActivityUpdater.java

```diff
@@ -30,9 +34,16 @@ public final class HelpThreadActivityUpdater implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadActivityUpdater.class);
     private static final int SCHEDULE_MINUTES = 30;
     private static final int ACTIVITY_DETERMINE_MESSAGE_LIMIT = 11;
+    private static final int PERSIST_DURATION_VALUE = 12;
+    private static final ChronoUnit PERSIST_DURATION_UNIT = ChronoUnit.HOURS;
```

Comment:
persist what? the name should clarify that

Reply:
done.